include/ola/io/Serial.h

```diff
@@ -60,21 +60,27 @@ typedef enum {
 bool UIntToSpeedT(uint32_t value, speed_t *output);
 
 /**
- * @brief Try to open the path, respecting UUCP locking.
+ * @brief Try to open the path and obtain a lock to control access
  * @param path the path to open
  * @param oflag flags passed to open
  * @param[out] fd a pointer to the fd which is returned.
  * @returns true if the open succeeded, false otherwise.
  *
+ * Depending on the compile-time configuration, this will use either flock()
+ * (the default) or UUCP locking.  See: ./configure --enable-uucp-locking.
+ *
  * This fails-fast, it we can't get the lock immediately, we'll return false.
  */
-bool AcquireUUCPLockAndOpen(const std::string &path, int oflag, int *fd);
+bool AcquireLockAndOpen(const std::string &path, int oflag, int *fd);
```

Comment:
Please leave the other one documented, but deprecate it and @see the new function.

Reply:
Ok, done.  The two pairs of routines (lock/unlock) now "@see" each other, and the two UUCP versions "@deprecated" to the corresponding new routines.